packages/fetch-api/README.md

```diff
@@ -152,6 +152,40 @@ emoji(options?: PaginationOptions): Promise<GifsResult>
 const { data: gifs } = await gf.emoji()
 ```
 
+## _emoji default variations_
+
+Returns a list of all the default emoji variations.
+
+##### Signature:
+
+```typescript
+emojiDefaultVariations(options?: PaginationOptions): Promise<GifsResult>
+```
+
+> Options: [Pagination Options](#pagination-options)
+
+##### Example:
+
+```typescript
+const { data: gifs } = await gf.emojiDefaultVariations()
```

Comment:
@pshoniuk can this just be part of `emojiVariations`? 

```typescript
// no argument gives you the defaults 
const { data: gifs } = await gf.emojiVariations()

Reply:
@giannif I don't think so, `emojiDefaultVariations` works more like the old emoji endpoint and returns Gis with extra details. On the other hand, `emojiVariations` returns variations for a particular gif from the result of the first endpoint and doesn't support pagination. I think that mixing this logic would not be a good idea. I borrowed the names of these API calls from the mobile SDK for consistency, but we can rename `emojiDefaultVariations` to emphasize the difference with `emojiVariations`

Reply:
@pshoniuk okay, I haven't been keeping up with what these new endpoints actually are. Is there docs on the mobile SDK on these I could read?